tokens/custom/global-vars.css

```diff
@@ -61,8 +61,11 @@
 	--spectrum-examples-gradient: linear-gradient(45deg, var(--spectrum-magenta-1500), var(--spectrum-blue-1500));
 
 	/* Gradients that do not change with the color theme, for use in static color backgrounds. */
-	--spectrum-examples-gradient-static-black: linear-gradient(45deg, rgb(255, 241, 246), rgb(238, 245, 255));
-	--spectrum-examples-gradient-static-white: linear-gradient(45deg, rgb(64, 0, 22), rgb(14, 24, 67));
+	--spectrum-examples-gradient-static-black: linear-gradient(45deg, rgb(255 241 246), rgb(238 245 255));
+	--spectrum-examples-gradient-static-white: linear-gradient(45deg, rgb(64 0 22), rgb(14 24 67));
 
 	--spectrum-coach-indicator-ring-static-white-color: var(--spectrum-white);
+
+	/* override for Spectrum Tokens value until a Style Dictionary transform gets written for our usage */
+	--spectrum-corner-radius-1000: 9999px;
```

Comment:
This adds the pill corner rounding as a custom property. I can ask Garth about that current value of `corner-radius-1000`, and dig into our broken corner rounding guidance page.